src/site/content/en/react/how-amp-can-guarantee-fastness-in-your-nextjs-app/index.md

```diff
@@ -0,0 +1,99 @@
+---
+layout: post
+title: How AMP can guarantee fastness in your Next.js app
+authors:
+  - houssein
+subhead: |
+  Learn about the benefits and tradeoffs of supporting AMP in your Next.js app
+date: 2019-10-17
+draft: true
+codelabs: how-to-use-amp-in-nextjs
+---
```

Comment:
I realized that this doc was a mixture between a conceptual overview, guidance, and hands-on tutorial. I moved the hands-on tutorial to a codelab. This also solved the word count problem. Each doc is nice and small now.